srai/datasets/_base.py

```diff
@@ -113,12 +113,13 @@ def train_test_split_bucket_regression(
             regionalizer = H3Regionalizer(resolution=resolution)
             regions = regionalizer.transform(gdf)
             joined_gdf = gpd.sjoin(gdf, regions, how="left", predicate="within")  # noqa: E501
+            joined_gdf.rename(columns={"index_right": "h3_index"}, inplace=True)
 
-            averages_hex = joined_gdf.groupby("region_id").size().reset_index(name=target_column)
```

Comment:
Instead, I would set the index name in the loaded dataset to `"region_id"` to keep it aligned with the rest of our loaders.

Reply:
I meant `"feature_id"` of course ;)